### PR TITLE
Fixed bug in large gutter modifier

### PR DESCRIPTION
**Pull request by jimmynotjim**
Fixed a bug where columns nested 2 levels deep in a large-gutter modifier gain the extra wide gutters.

## Changes

- Limited extra wide gutters to direct children of the large-gutter modifier

## Testing

Test this markup in a project

```
             <div class="content-l
                                content-l__large-gutters">
                <section class="content-l_col
                                content-l_col-1-2">
                    <h2>Look, I'm 1/2 the page!</h2>
                    <div class="content-l">
                        <div class="content-l_col
                                    content-l_col-1-2">
                            I'm a 1/4 column
                        </div>
                        <div class="content-l_col
                                    content-l_col-1-2">
                            I'm a 1/4 column
                        </div>
                    </div>
                </section>

                <section class="content-l_col
                                content-l_col-1-2">
                    <h2>I'm also 1/2 the page</h2>
                    <div class="content-l">
                        <div class="content-l_col
                                    content-l_col-1-2">
                            I'm a 1/4 column
                        </div>
                        <div class="content-l_col
                                    content-l_col-1-2">
                            I'm a 1/4 column
                        </div>
                    </div>
                </section>
            </div>
```

## Review

- @Scotchester 
- @KimberlyMunoz 
- @cfarm
- @contolini 

## Screenshots

__Before:__

screen shot 2016-02-03 at 2 10 31 pm

__After:__

screen shot 2016-02-03 at 2 09 50 pm

## Todos

- None

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-layout/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-layout",
-  "version": "3.0.3",
+  "version": "3.0.4",
```
Comment (contolini) [inline src/cf-layout/package.json]: We've been oscillating on the topic but I think we've finally agreed to not update version numbers in PRs. See https://github.com/cfpb/capital-framework/pull/267#discussion_r51621665.

So just leave this at v3.0.3.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- Fixed a bug where columns nested 2 levels deep in a large-gutter modifier gain the extra wide gutters.
```
Comment (contolini) [inline CHANGELOG.md]: Stick `**cf-layout:** ` in front of the change to indicate the changed component (see [previous examples](https://raw.githubusercontent.com/cfpb/capital-framework/canary/CHANGELOG.md)).

**Comment by contolini** [comment]
:+1: Nice catch!

File: CHANGELOG.md (hunk 2 above)
Comment (jimmynotjim) [inline CHANGELOG.md]: I should have looked at your PR first :D. Fixed

**Comment by jimmynotjim** [comment]
@contolini Looking at your PR, it doesn't mention bugfixes being branched/PR'd against `master` like your issue does. What's the proper way here? Should I close this and open it against `canary`?

**Comment by contolini** [comment]
@jimmynotjim good call. Leave this PR how it is because we'll be doing a release within the next couple of days. Plus this isn't a "OMG EVERYTHING IS BROKEN" type of bug. I'll add some guidance to the contributing docs.

**Comment by Scotchester** [comment]
:+1:

**Pull request merged**